suite-native/module-accounts/src/components/TransactionListHeader.tsx

```diff
@@ -32,6 +36,32 @@ type AccountsNavigationProps = TabToStackCompositeNavigationProp<
     AccountsStackParamList
 >;
 
+const GraphlessHeader = ({ accountKey }: { accountKey: AccountKey }) => {
```

Comment:
```suggestion
const TestnetAccountTransactionListHeader = ({ accountKey }: { accountKey: AccountKey }) => {
```

Reply:
The component was renamed and moved to separate  file in f5d488e1f.